.github/workflows/build-test.yml

```diff
@@ -21,6 +21,7 @@ jobs:
       - uses: actions/checkout@v2
         with:
           submodules: true
+      - uses: arduino/setup-protoc@v3
```

Comment:
What is this? what is it used for?

Reply:
To add `protoc` binary into the path on `linux/mac/windows` environments (used to compile the `.proto` files).